src/js/vic-v2/config/form.js

```diff
@@ -4,6 +4,7 @@ import fullSchemaVIC from 'vets-json-schema/dist/VIC-schema.json';
 
 import IntroductionPage from '../components/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
+// import PhotoField from '../components/PhotoField';
```

Comment:
Should this be removed? Or is this to promote visibility of what is going on elsewhere with the asyncLoader?

Reply:
This is I think a bad merge. None of the changes in this file should be in here.